deploy/helm/sumologic/values.yaml

```diff
@@ -85,6 +85,10 @@ sumologic:
   # If enabled, collect K8s events
   eventCollectionEnabled: true
 
+  events:
+    # Source category for the Events source. Default: "{clusterName}/events"
+    sourceCategory: ""
```

Comment:
why not use what we had below to set the default `sourceCategory: "%{namespace}/%{pod_name}"`

Reply:
I am using the source category we had for events before for consistency. Also what would pod name be in this case? For logs it's the name of the pod that logs are coming from I assume, but events are not only about pods. We can't use the pod name of the events pod itself either, since we don't know the pod name at the time of the source creation in TF